Update Server link to GitHub repo - Totango (#4822)
segmentio/segment-docs: src/connections/destinations/catalog/totango/index.md

title: Totango Destination
id: 54521fdb25e721e32a72eefa
---

> Our Totango destination code is all open-source on GitHub if you want to check it out: [JavaScript](https://github.com/segment-integrations/analytics.js-integration-totango)

Reviewer: ```suggestion
Segment's Totango destination code is all open-source on GitHub: [JavaScript](https://github.com/segment-integrations/analytics.js-integration-totango){:target="_blank"}.
```

Reviewer: Thanks for submitting @bcaudillo. I'm still seeing a `404` error with the link you updated.

Author: @rchinn-segment I just corrected the link, sorry about that.
